### PR TITLE
pluma-view: Fix: unexpectedly switch between tabs

### PR DESCRIPTION
**Pull request by sc0w**
Closes https://github.com/mate-desktop/pluma/issues/349

### CODE REVIEW

**Comment by lukefromdc** [comment]
I do not have the sort of hardware required to test this. My mouse only has vertical scroll, and I do not know how to use any kind of touchpad gestures on a laptop.

**Comment by sc0w** [comment]
The bug is very weird...

**Comment by sc0w** [comment]
I will wait the tests of the reporter, I will tell you here when the PR is ready.

**Comment by sc0w** [comment]
The reporter says latest changes fixes the issue, @vkareh please test again.

If you don't have the required hardware to reproduce the issue, just test if all is working as before with the scroll in the view.

**Comment by raveit65** [comment]
I don't understand the issue, sadly too much information is written in report, but no clear bug description in 2 or 3 sentences ;)
With current pluma from master 2-finger-scrolling (v/h) with touchpad works for me in document.
If cursor pointer is over the tabs than using 2-finger-scrolling (v/h) do switch active tabs.
So what's the problem?
Last but not least, i hate using the touchpad, and normally it is disabled for my notebook.
So i am not really an expert with using it :smile:  
But i am happy to test PR if i know what should i test.

**Comment by sc0w** [comment]
@raveit65 I can confirm the issue with modifications in the code

In the view, with the events  `GDK_SCROLL_UP GDK_SCROLL_DOWN GDK_SCROLL_LEFT or GDK_SCROLL_RIGHT`, the tabs are swithing, like the video:  https://github.com/mate-desktop/pluma/issues/349#issuecomment-413572946

You need to reproduce the issue this hardware: https://github.com/mate-desktop/pluma/issues/349#issuecomment-408611270 or modify the code manually to force that events over the view.

With normal hardware, only the event `GDK_SCROLL_SMOOTH` is working over the view and there is no issues.

Just test if all is working like before.

:)

File: pluma/pluma-view.c [hunk 1]
```diff
@@ -181,6 +183,36 @@ document_read_only_notify_handler (PlumaDocument *document,
 				    !pluma_document_get_readonly (document));
 }
 
+static gboolean
+pluma_view_scroll_event (GtkWidget      *widget,
+			 GdkEventScroll *event)
```
Comment (raveit65) [inline pluma/pluma-view.c]: indentation?

Comment (sc0w) [reply pluma/pluma-view.c]: It looks fine here: https://raw.githubusercontent.com/mate-desktop/pluma/1cd757ab5b2688f9628bd97d1a634020b544379e/pluma/pluma-view.c

The tabs... A lot of functions have the same indentation.

I will fix it later with only spaces.

**Comment by raveit65** [comment]
i used `git format-patch ca00615b510549a4abfe44f7b1f97485e1e9f804`
to generate the patch.
And patch looks like this with pluma.
https://www.dropbox.com/s/jst6r9sosumi49c/pluma_0001-pluma-view-Fix-unexpectedly-switch-between-tabs.patch?dl=0

Probably a problem with different tab size in editors from people.
You an avoid this with using only spaces in new code especial for such a line
```
+static gboolean
+pluma_view_scroll_event (GtkWidget      *widget,
+            GdkEventScroll *event)
+{
```

**Pull request merged**